booking/src/main/java/org/dcsa/conformance/standards/booking/action/BookingAction.java

```diff
@@ -125,6 +128,12 @@ protected void updateDSPFromResponsePayload(ConformanceExchange exchange) {
     }
   }
 
+  private String getCbrFromNotificationPayload(JsonNode requestJsonNode) {
+    if(requestJsonNode.path("data").path("carrierBookingReference").asText(null) != null) {
+      return requestJsonNode.path("data").path("carrierBookingReference").asText(null);
+    }
+    return null;
+  }
```

Comment:
Seems overly complex; it looks like this could just be `return requestJsonNode.path("data").path("carrierBookingReference").asText(null);`

Reply:
done